pom.xml

```diff
@@ -1861,6 +1861,17 @@
                             <fileset>
                                 <directory>${project.basedir}/web-ui/dist</directory>
                             </fileset>
+                            <fileset>
+                                <!-- Clean up the extension spark jars, to avoid include unspecified version -->
```

Comment:
```suggestion
                                <!-- clean up all versions of Spark extension, to avoid including outdated artifacts in binary distribution -->
```